akka-remote/src/main/scala/akka/remote/artery/BufferPool.scala

```diff
@@ -93,7 +88,7 @@ private[remote] object EnvelopeBuffer {
   val UsAscii = Charset.forName("US-ASCII")
```

Comment:
The next two `val`s are not needed but they broke binary compatibility, I would like very much to remove them.